src/common/components/hosts/HostRowDetail.tsx

```diff
@@ -120,6 +126,8 @@ export const HostDetail = ({
   onDiskRole,
   AdditionalNTPSourcesDialogToggleComponent,
   hideNTPStatus = false,
+  showStorage = true,
```

Comment:
The `showStorage` prop has been removed, it should not appear anymore.
This looks like a conflict issue.

Reply:
Yes, is a conflict issue